src/interfaces.ts

```diff
@@ -156,7 +156,7 @@ export interface LoggerEvents {
   onStop: StopReason;
 }
 
-export type StopReason = '401' | '404' | 'force';
+export type StopReason = 401 | 404 | 'force';
```

Comment:
It might be overkill but I wonder if we should go ahead and add the codes as strings here too. Do we know why these changed from strings to numbers?

Reply:
I don't think we have a definitive reason just yet... At least from what we know thus far.  I think the working theory which makes a bit of sense to me is that when we replaced Super Agent with Axios, it updated the structure but we either didn't realize that was the case at first or just didn't think it would affect Client Logger.  Though definitely not 100% sure of this.

As for checking the strings and the number, should we also check both `err.respone.status` and `err.status`?

Reply:
Yeah but we migrated from superagent quite awhile ago so it seems weird that this would only become a problem now but it is possible. Yeah, it might be overkill and unnecessary but I'd rather be safe than sorry.